### PR TITLE
refactor: 💡 Migrate SQFormMaskedReadOnlyField to TypeScript

### PR DESCRIPTION
**Pull request by aharpt**
✅ Closes: #474

### CODE REVIEW

File: src/components/SQForm/SQFormMaskedReadOnlyField.tsx [hunk 1]
```diff
@@ -1,16 +1,41 @@
...
+  /** Valid mask array; custom or from utils/masks.js */
+  mask?: maskProp;
+  /** Name of the field will be the Object key of the key/value pair form payload */
+  name: string;
+  /** Descriptive label of the input */
+  label: string;
+  /** Placeholder text used inside the input field to provide hints to the user */
+  placeholder?: string;
+  /** Size of the input given full-width is 12. */
+  size?: 'auto' | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12;
```
Comment (20BBrown14) [inline src/components/SQForm/SQFormMaskedReadOnlyField.tsx:24]: SQFormReadOnlyFieldProps extends BaseFieldProps which already defines `name`, `label`, and `size` for us so no need to do it here.

Comment (aharpt) [reply src/components/SQForm/SQFormMaskedReadOnlyField.tsx:24]: @20BBrown14 Ahh I see, thanks for pointing that out.  Will update my PR.

**Comment by 20BBrown14** [comment]
Now that the issue/244 branch is up to date with master and has a good build, you should be able to rebase your branch and fix the failing tests that we're seeing in the CI

**Comment by aharpt** [comment]
Closing this PR without merging since something weird happened with git rebase.

**Pull request closed without merging**